### PR TITLE
Branching for 0.8.1 release.

### PR DESCRIPTION
**Pull request by vgapeyev**
Note a thing that, I think, has to be done slightly differently: 

- The v0.8.1 branch is not from the tip of main, since there are already a few commits that we do not want included.  Consequently, this PR will not be merged right away -- the Maven and GitHub release steps will be done from the branch. 

## License Information

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<!-- Template: after a release, copy and paste out below
```
Comment (johnedquinn) [inline CHANGELOG.md:7]: You can remove this.

Comment (vgapeyev) [reply CHANGELOG.md:7]: Oh, you don't like my wonderful invention? 

This does not render in HTML. 
When starting a new dev cycle, it saves one the task of creating fresh headings for the log sections, either by typing them in (risking typos) or by copy-pasting each individual line.  Now one can cut-and-paste the whole block and be done with  it.

Comment (vgapeyev) [reply CHANGELOG.md:7]: Or did you mean to merely remove the text `Template: .... below`?

**Review by johnedquinn — DISMISSED** [review]

**Review by johnedquinn — DISMISSED** [review]
Just need to resolve conflicts. But looks good.

**Pull request merged**